app/views/videos/show.html.erb

```diff
@@ -0,0 +1 @@
+show
```

Comment:
Refer to previous comment about putting this in a separate pull request.

Also, add a linebreak at the end of this file.